manifests/config.pp

```diff
@@ -39,6 +39,7 @@
   $send_emails = false,
   $use_checkin_barcode = true,
   $badge_promo_codes_enabled = false,
+  $prereg_request_hotel_info_duration = false,
```

Comment:
Hm, we typically only set boolean values on boolean config vars. I'd probably set this to '0' or '-1' or something.